README.md

```diff
@@ -73,18 +73,25 @@ In a terminal window run these commands to run the jest test suite in watch mode
 
 ```sh
 $ npm -i -g jest
```

Comment:
Can you remove this global install as well? It is not needed.

Reply:
Done